pylib/System/TestDef.py

```diff
@@ -218,6 +218,7 @@ def parseOptions(self, log, options, keyvals, target):
             log['status'] = 1
             # pass the errors back
             log['stderr'] = stderr
+            import pdb; pdb.set_trace()
```

Comment:
Just curious - what does this do? Probably want the import at the top with the others, and then a comment here that explains what this command is intending to do.